pkg/generator/util.go

```diff
@@ -1,7 +1,25 @@
 package generator
 
 // toPlural makes "input" word plural.
-// TODO: make this more grammatically correct for special nouns.
+// TODO: make this an input parameter as English grammar is highly variable
 func toPlural(input string) string {
+	lastchar := input[len(input)-1:]
+
+	if lastchar == "s" {
+		return input + "es"
+	} else if lastchar == "x" {
```

Comment:
return `input + "es"` is common for 4 statements, you could refactor  : 
```
lastchar := input[len(input)-1:]
lasttwo := input[len(input)-2:]

if lastchar == "s" || 
  lastchar == "x"  || 
  lasttwo == "ch" ||
  lasttwo == "sh" {
  return input + "es"
} else if lastchar == "y" {
  return input[0:len(input)-2] + "ies"
}

return input + "s"
```

Reply:
cc @hasbro17 I think this boils down to a style choice, what do you think?

Reply:
Both are fine, although I think the existing block is readable enough.